src/App.php

```diff
@@ -114,11 +114,18 @@ protected function loadSingle(Config $config, array $projectDefinition, string $
         }
     }
 
+    public function getEnabledTables(Config $config) : array
```

Comment:
Ještě prosím tě unit test

Reply:
nasmerujes ma ako to urobit? z pohladu do AppTest mi neni jasne preco tam je assertCount na 5 ked v config.json su 3 tabulky..

Reply:
Třeba takhle? https://gist.github.com/JakubMatejka/f91a0f94d8970636766e606fd7e6ef6e#file-apptest-php-L71 Co se týče těch pěti datasetů, tak to budou nejspíš časový dimenze (je to hack, který tam dodává Writer automaticky).

Reply:
aha, ja som povodne chcel len do config.json pridat jednu disabled tabulku a vlastne v testoch by sa nic nezmenilo ale moze byt aj takto ako navrhujes.

Reply:
Anebo ;o)